### PR TITLE
Add new versions of rpmostree and ostree packages

### PR DESCRIPTION
**Pull request by suezzelur**

### CODE REVIEW

File: SPECS/libhif/libhif.spec [hunk 1]
```diff
@@ -36,16 +39,20 @@ used by PackageKit and rpm-ostree.
 %package devel
 Summary: GLib Libraries and headers for libhif
 Requires: libhif
+Provides: pkgconfig(libhif)
+Provides: pkgconfig(hawkey)
+Provides: pkgconfig(rpm)
```
Comment (toliaqat) [inline SPECS/libhif/libhif.spec]: This packages should not Provide pkgconfig(rpm) and pkgconfig(hawkey) etc. They should be provided by respective packages SPEC files.

File: SPECS/ostree/ostree.spec [hunk 2]
```diff
@@ -1,25 +1,34 @@
...
+Requires:       systemd
 BuildRequires:	attr
 BuildRequires:	python2-libs
 BuildRequires:	python2
 BuildRequires:	gobject-introspection
+BuildRequires:	gobject-introspection-devel
 BuildRequires:	gobject-introspection-python
+BuildRequires:  gpgme-devel
+BuildRequires:  libcap
+BuildRequires:  systemd
```
Comment (toliaqat) [inline SPECS/ostree/ostree.spec]: Should we have BuildRequire of dracut and mkinitcpio here?

File: SPECS/rpm-ostree/rpm-ostree.spec [hunk 3]
```diff
@@ -8,44 +8,72 @@ URL:            https://github.com/cgwalters/rpm-ostree
 Vendor:		VMware, Inc.
 Distribution:	Photon
 # We always run autogen.sh
-BuildRequires: autoconf automake libtool
+BuildRequires: autoconf automake libtool git
```
Comment (toliaqat) [inline SPECS/rpm-ostree/rpm-ostree.spec]: Its OK for now but make these BuildRequires: multiline next time

File: SPECS/rpm-ostree/rpm-ostree.spec [hunk 4]
```diff
@@ -8,44 +8,72 @@ URL:            https://github.com/cgwalters/rpm-ostree
...
+
 %prep
-%setup -q -n %{name}-%{version}
+%autosetup -Sgit -n %{name}-%{version}
+git clone git://git.gnome.org/libglnx libglnx
 
 %build
 env NOCONFIGURE=1 ./autogen.sh
-%configure --disable-silent-rules --enable-patched-hawkey-and-libsolv --enable-usrbinatomic
+%configure --disable-silent-rules --enable-usrbinatomic
```
Comment (toliaqat) [inline SPECS/rpm-ostree/rpm-ostree.spec]: Is hawkey and librepo being automatically picked up now?

File: installer/package_list.json [hunk 5]
```diff
@@ -61,7 +61,7 @@
                 "cmake", "gpgme-devel", "parted", "vim", "xz-devel", "xerces-c-devel", "xml-security-c-devel", 
                 "nss-devel", "ncurses-devel", "tzdata", "itstool", "sed-lang", "gptfdisk", "ntp", "flex", "patch", "m4", "dracut", "dracut-tools", 
                 "libarchive", "libsoup", "asciidoc", "mkinitcpio",
-                "ostree", "tdnf-devel", "rpm-ostree", "createrepo", "glibmm", "deltarpm", "cdrkit", "dparted", "libgsystem",
```
Comment (toliaqat) [inline installer/package_list.json]: Add libhif in input.json file as well.

File: SPECS/rpm-ostree/rpm-ostree.spec (hunk 4 above)
Comment (suezzelur) [inline SPECS/rpm-ostree/rpm-ostree.spec]: This is being reported as an invalid configuration option now. rpm-ostree does not look like needs patched versions anymore.

File: SPECS/ostree/ostree.spec (hunk 2 above)
Comment (suezzelur) [inline SPECS/ostree/ostree.spec]: It does not seem to require that. There are only a few scripts and conf files getting added to the package here.

**Comment by toliaqat** [comment]
Merged, Thanks.

**Pull request closed without merging**